packages/bpk-docs/src/pages/DialogsPage/DialogsPage.js

```diff
@@ -0,0 +1,150 @@
+  constructor() {
+    super();
+
+    this.onOpen = this.onOpen.bind(this);
+    this.onClose = this.onClose.bind(this);
+
+    this.state = {
+      isOpen: false,
+    };
+  }
+
+  onOpen() {
+    this.setState({
+      isOpen: true,
+    });
+  }
+
+  onClose() {
+    this.setState({
+      isOpen: false,
+    });
+  }
+
+  render() {
+    const { children, ...rest } = this.props;
+
+    return (
+      <div>
+        <BpkButton onClick={this.onOpen}>Open Dialog</BpkButton>
+        <BpkDialog
+          closeLabel="Close dialog"
+          id="my-dialog"
+          className="my-classname"
+          isOpen={this.state.isOpen}
+          onClose={this.onClose}
+          renderTarget={() => document.getElementById('application-container')}
+          getApplicationElement={() => document.getElementById('portal-target')}
+          {...rest}
+        >
+          <div>{children}</div>
+          <BpkButtonLink onClick={this.onClose}>Close dialog</BpkButtonLink>
+        </BpkDialog>
+      </div>
+    );
+  }
+}
+
+DialogContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default dialog',
+    blurb: [
+      <Paragraph>
+        The default has a title and close icon. Tapping the scrim or close icon
+        will dismiss the dialog.
+      </Paragraph>,
+    ],
+    examples: [
+      <DialogContainer title="Default Dialog.">
+        <Paragraph>You can put anything you want in here.</Paragraph>
+      </DialogContainer>,
+    ],
+  },
+  {
+    id: 'non-dismissible',
+    title: 'Non dismissible',
+    blurb: [
+      <Paragraph>
+        In cases when you want the user to explicitly make or accept the choice,
+        this configuration means the dialog has no close icon, and tapping the
+        scrim will NOT dismiss it. Instead, the user must interact with the
+        dialogs content in order to dismiss it.
+      </Paragraph>,
+    ],
+    examples: [
+      <DialogContainer dismissible={false}>
+        <Paragraph>You can put anything you want in here.</Paragraph>
+      </DialogContainer>,
+    ],
+  },
+];
+
+const DialogsPage = () => (
+  <DocsPageBuilder
+    title="Dialogs"
+    blurb={[
+      <Paragraph>
+        Dialogs inform users about a specific task and may contain critical
+        information, or require decisions or acknowledgement. For example:
+      </Paragraph>,
+      <ul>
+        <li>
+          Alert dialogs: urgent interruptions which tell users about a situation
+          and require acknowledgement.
+        </li>
+        <li>
+          Decision dialogs: which require users to confirm or make a choice.
+        </li>
+      </ul>,
+      <br />,
+      <Paragraph>
+        Like modals, when triggered, dialogs will emerge from the centre of the
+        viewport with a backdrop to indicate their separation from everything
+        else and retain focus until dismissed or a required action has been
```

Comment:
This is quite a long sentence. I think it would be better split into two, like:

> Like modals, when triggered, dialogs will emerge from the centre of the viewport with a backdrop to indicate their separation from everything else. They retain focus until dismissed or a required action has been taken.